README.md

```diff
@@ -125,6 +125,155 @@ record = client.write(record_type, data)
 print 'Wrote record {0}'.format(record.meta.record_id)
 ```
 
+## Searching records
+
+E3DB supports complex search options for finding records based on the fields stored in record metadata. 
+
+For example, to list all records of type `contact` and print a simple report containing names and phone numbers:
+
+```python
+# setup
+import e3db
+from e3db import Search
+client = e3db.Client(' config ')
+
+query = Search(include_data=True).Match(record_type=['contact'])
+results = client.Search(query)
+
+for record in results:
+    full_name = "{0} --- {1}".format(record.data['first_name'], record.data['last_name'])
+    print "{0} --- {1}".format(full_name, record.data['phone'])
+```
+
+The full list of parameters you can search for are under `e3db.types.Params`. Searching gives you access to chaining more powerful queries such as conditional operators, exlusions, and date filters.     
+
+To search for records of type `season` that have values `summer` and `sunny`, create the following query:
```

Comment:
I think we need some explanatory text/examples around what a record "having values" means, are you talking about records with metadata that has keys of "summer" and "sunny"?